sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOContentSecurityPolicy.md

```diff
@@ -0,0 +1,74 @@
+---
+external help file:
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/remove-spocontentsecuritypolicy
+applicable: SharePoint Online
+title: Remove-SPOContentSecurityPolicy
+schema: 2.0.0
+author: jaobrie
+ms.author: jaobrie
+manager: ryannak
+ms.reviewer:
+---
+
+# Remove-SPOContentSecurityPolicy
+
+## SYNOPSIS
+
+Adds an entry to the Content Security Policy configuration.
```

Comment:
I think this should say "remove an entry from the Content Security Policy configuration",right?

Reply:
I would add: "in multi geo environments, Content Security Policy entries are different for each geo" (or something on that line)